tests/test_claim_upload_material_bridge.py

```
@@ -0,0 +1,126 @@
+        assert "ТРЕБОВАНИЕ ИЗ ДОКУМЕНТА:" in prompt
+        assert extracted.important_facts == [
+            "ТРЕБОВАНИЕ ИЗ ДОКУМЕНТА: вернуть 500 000 тенге"
+        ]
+        assert "ТРЕБОВАНИЕ ИЗ ДОКУМЕНТА" in extracted.as_context()
+
+    asyncio.run(scenario())
+
+
+def test_empty_claim_prayer_is_recovered_from_uploaded_pretrial(monkeypatch) -> None:
+    async def scenario() -> None:
+        async def original_draft(_self, _context, _research, language="ru"):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `original_draft`

(ANN202)